bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -973,13 +832,17 @@ spec:
             --metrics-bind-address "127.0.0.1:29103" \
```

Comment:
so the same container port is going to be used to scrape metrics from both node and controller threads right ? since its same process?